### PR TITLE
SC-6545 – Shorten button text and add download icon

### PR DESCRIPTION
**Pull request by monaweitzenberg**
# Description

## Links to Tickets or other pull requests
https://ticketsystem.schul-cloud.org/browse/SC-6545

## Changes
update button-text from 'Alle Dateien unbewerteter Hausaufgaben herunterladen' into 'Unbewertete Hausaufgaben' and add a download-icon on the button

## Screenshots of UI changes
image
image

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: views/homework/components/bulk-grading.hbs [hunk 1]
```diff
@@ -7,6 +7,7 @@
 				{{#if @root.ungradedFileSubmissions.empty}}disabled="disabled" {{/if}}
 				data-files="{{@root.ungradedFileSubmissions.urls}}"
 			>
+			<i class="fa fa-cloud-download"></i>
```
Comment (DanielRieckhof) [inline views/homework/components/bulk-grading.hbs:10]: Maybe lint will complain on missing spaces/tabs

**Review by ABialoskorski — APPROVED** [review]

**Review by DanielRieckhof — APPROVED** [review]

**Review by seulki-lee-cloud — CHANGES_REQUESTED** [review]
Compared to the suggestion image from the ticket (https://ticketsystem.schul-cloud.org/browse/SC-6545).
The color of the icon and text in the button is so light grey. It looks disabled button.
Please check the mona's suggestion image again, and use the right dark grey color.
Thank you :)

**Review by seulki-lee-cloud — COMMENTED** [review]
Thank you

**Comment by seulki-lee-cloud** [comment]
thank you

**Review by seulki-lee-cloud — APPROVED** [review]
Thank you

**Pull request merged**